modules/matterhorn-index-service/src/main/java/org/opencastproject/index/service/catalog/adapter/events/CommonEventCatalogUIAdapter.java

```diff
@@ -77,8 +81,24 @@ public Catalog storeFields(MediaPackage mediaPackage, MetadataCollection abstrac
     }
 
     MetadataField<?> series = abstractMetadata.getOutputFields().get(DublinCore.PROPERTY_IS_PART_OF.getLocalName());
-    if (series != null && series.isUpdated() && isNotBlank(series.getValue().get().toString()))
-      mediaPackage.setSeries(series.getValue().get().toString());
+    if (series != null && series.isUpdated() && isNotBlank(series.getValue().get().toString())) {
+      String seriesID = series.getValue().get().toString();
+      mediaPackage.setSeries(seriesID);
+      try {
+        DublinCore seriesDC = getSeriesService().getSeries(seriesID);
```

Comment:
This is for sure the safty way to get the series title. But why not grab it from the given collection?

Reply:
IIRC it was because the series were explicitly not parsed from the json...